boards/cats_rev1Pro/Core/Src/tasks/task_imu_read.c

```diff
@@ -57,13 +60,13 @@ void task_imu_read(void *argument) {
     //            2, gyroscope[0], gyroscope[1], gyroscope[2],
     //            acceleration[0], acceleration[1], acceleration[2], temperature);
 
-    mmc5983ma_read_calibrated(&MAG, data);
-    magneto_data.magneto_x = data[0];
-    magneto_data.magneto_y = data[1];
-    magneto_data.magneto_z = data[2];
-    magneto_data.ts = osKernelGetTickCount();
+    mmc5983ma_read_calibrated(&MAG, mag_field);
+    global_magneto.magneto_x = mag_field[0];
+    global_magneto.magneto_y = mag_field[1];
+    global_magneto.magneto_z = mag_field[2];
+    global_magneto.ts = osKernelGetTickCount();
 
-    global_magneto = magneto_data;
```

Comment:
Somehow the recording of the magnetometer is gone... was it never pushed to main?

Reply:
It was I just forgot to pull, we now have some merge conflicts. I will try to resolve it somehow